models/db.js

```diff
@@ -33,6 +33,18 @@ db.otpModel = require('../models/otp')(sequelize, Sequelize);
 db.categoryModel = require('../models/category')(sequelize, Sequelize);
 db.productModel = require('../models/product')(sequelize, Sequelize);
 db.productImageModel = require('../models/product_image')(sequelize, Sequelize);
+db.cartModel = require('../models/cart')(sequelize, Sequelize);
+
+db.authModel.hasMany(db.productModel, { foreignKey: 'user_id' });
+db.authModel.hasMany(db.cartModel, { foreignKey: 'user_id' });
+
+db.productModel.belongsTo(db.authModel, { foreignKey: 'user_id' });
+db.productModel.hasMany(db.cartModel, { foreignKey: 'product_id' });
+
+db.cartModel.belongsTo(db.authModel, { foreignKey: 'user_id' });
+db.cartModel.belongsTo(db.productModel, { foreignKey: 'product_id' });
+
+db.productModel.hasMany(db.productImageModel, { foreignKey: 'product_id' });
```

Comment:
same as here 
why you put product and product image association in cart module?

Reply:
product and product_image association are defined because at the time of view cart the information of product with its product image is displayed for that association is required